/*
Vapor Test Suite
2000: Basic throw and catch

- output
Hello exception world.
*/

using Core;

class MyException: Exception
{
	public String message;

	public (String message)
	{
		this.message = message;
	}
}

class Test
{
	public static Main()
	{
		try
		{
			throw new MyException("Hello exception world.");
		}
		catch (MyException e)
		{
			Console.WriteLine(e.message);
		}
	}
}
